modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,3 +1,12 @@
+resource "aws_db_subnet_group" "database" {
+  name       = "${var.infrastructurename}-vpc"

```

Comment:
Suggestion: Use a more self-descriptive suffix like `-dbsubnet` instead of `-vpc`

Reply:
This is done to align with previous naming convention (before db subnet migration) and is aligned with migration doc. If you follow the steps, there will be no re-deployment of Simphera resources, only possibly update in-place.